frontend/src/features/admin-form/settings/components/FormEmailSection.tsx

```diff
@@ -143,18 +147,22 @@ export const FormEmailSection = ({
             isRequired={isEmailMode}
             useMarkdownForDescription
             description={DESCRIPTION_TEXT}
+            isHighContrast={isHighContrast}
           >
             {t(
               'features.adminForm.settings.emailNotifications.section.regular.label',
             )}
           </FormLabel>
-          <AdminEmailRecipientsInput onSubmit={handleSubmitEmails} />
+          <AdminEmailRecipientsInput
+            onSubmit={handleSubmitEmails}
+            isDisabled={isDisabled}
+          />
           <FormErrorMessage>{get(errors, 'emails.message')}</FormErrorMessage>
           {isEmpty(errors) ? (
             <FormLabel.Description
               color="secondary.400"
               mt="0.5rem"
-              opacity={isDisabled ? '0.3' : '1'}
+              opacity={'1'}
```

Comment:
nit: Since this is no longer a ternary operation, it an be simplified
```suggestion
              opacity='1'
```

Reply:
Thanks! removed this